core/common/src/main/java/alluxio/util/LogUtils.java

```diff
@@ -127,7 +127,7 @@ public static void warnWithException(Logger logger, String message, Object ...ar
       logger.debug(message, args);
     } else {
       if (args.length > 0 && args[args.length - 1] instanceof Throwable) {
-        args[args.length - 1] = ((Throwable) args[args.length - 1]).getMessage();
+        args[args.length - 1] = (args[args.length - 1]).toString();
```

Comment:
Won't this print out the entire stack trace of the exception?

Reply:
No, this just prints `<ExceptionClass>: <Exception message>`. Before we would lose the exception type.